components/automate-gateway/api/compliance/reporting/reporting.proto

```diff
@@ -30,6 +30,21 @@ service ReportingService {
 			action: "compliance:reports:list"
 		};
 	};
+	// should cover /reportids
+    rpc ListReportIds(Query) returns (ReportIds) {
```

Comment:
can you add a comment here noting that this endpoint is *not* limited by the 10k es pagination thing

Reply:
done

Reply:
done